libs/isograph-react/src/tests/__isograph/Query/nodeField/entrypoint.ts

```diff
@@ -20,6 +20,7 @@ const normalizationAst: NormalizationAst = [
         { kind: "Variable", name: "id" },
       ],
     ],
+    concreteType: "Economist",
```

Comment:
This has been manually, changed from "Node" to "Economist"

Reply:
https://github.com/isographlabs/isograph/blob/e5ce3596bd385777f5d26c3cd36e5e59810a94d2/libs/isograph-react/src/tests/nodeQuery.ts#L6-L12

So there's no issue in rust code, this just can't be narrowed down without proper interface support